.goreleaser.yaml

```diff
@@ -22,6 +22,7 @@ builds:
       - -trimpath
     tags:
       - netgo
+      - embedassets
```

Comment:
I guess we need to build frontend too before go binaries ?

Reply:
https://github.com/grafana/phlare/pull/606/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R48 it's here i believe

Reply:
ho I see. yep